django/core/serializers/json.py

```diff
@@ -39,8 +39,7 @@ def end_serialization(self):
         if self.options.get("indent"):
             self.stream.write("\n")
         self.stream.write("]")
-        if self.options.get("indent"):
-            self.stream.write("\n")
+        self.stream.write("\n")
```

Comment:
One option we could consider would be if indent is set as 0 explicitly, we don't add the new line.
Then by default, a new line is added but if anyone really doesn't want it, they can get rid of it :thinking: 

I do however think it's fine. Jsonl always has new lines on the end for example

Reply:
I feel like we add a bit of extra branching for not much gain here. If the argument was named `minify` or something I'd be inclined to agree.